vespa-http-client/src/main/java/com/yahoo/vespa/http/client/config/FeedParams.java

```diff
@@ -246,11 +244,14 @@ public int getMaxChunkSizeBytes() {
             return maxChunkSizeBytes;
         }
 
-        public int getmaxInFlightRequests() {
+        public int getMaxInFlightRequests() {
```

Comment:
Same question as above re: public API

Reply:
This is a getter on the Builder. I don't think people use that so I'm willing to risk it  :-)